.ci/Jenkinsfile

```diff
@@ -47,6 +49,16 @@ pipeline {
                 sh './gradlew --no-daemon -s --no-build-cache clean build'
             }
         }
+        stage('Snyk Security Scan') {
+            when {
+                not {
+                    changeRequest()
+                }
+            }
```

Comment:
This will then run on any branch build, including manually triggered ones for misc branches. Is that correct?

Reply:
I would like to know that my PR is going to fix any reported security problems _before_ merging it, so running on branch builds sounds like a Good Idea... :thinking:.

Reply:
I think Chris is correct here, and also this is keeping consistent with release/6.0 & release/5.0
https://github.com/corda/corda-gradle-plugins/blob/master/.ci/Jenkinsfile
https://github.com/corda/corda-gradle-plugins/blob/release/5.0/.ci/Jenkinsfile